vars/folioexecuteDataMigrationUtils.groovy

```diff
@@ -138,9 +144,7 @@ def createTimeHtmlReport(tenantName, tenants, pgadminURL) {
                             modulesLongMigrationTime.put(moduleName, execTime)
                         }
                     }
-                    markup.bodyy(href: pgadminURL, target: "_blank") {
-                        builder.h2("pgAdmin")
-                    }
+
```

Comment:
It's not properly generating a report because of this URL, I'll add it additionally, if you remember last time we added this URL without tests because the whole pipeline didn't work